driver/driver-test-server/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <parent>
+        <groupId>org.neo4j.driver</groupId>
+        <artifactId>neo4j-driver-parent</artifactId>
+        <version>2.3-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.neo4j.driver</groupId>
+    <artifactId>neo4j-driver-test-server</artifactId>
+    <version>2.3-SNAPSHOT</version>
+
+    <packaging>jar</packaging>
+    <name>Neo4j Driver Test Server</name>
+    <description>Assembles and runs the Neo4j Data Protocol server</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.neo4j.ndp</groupId>
+            <artifactId>neo4j-ndp-transport-http</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.neo4j</groupId>
+            <artifactId>neo4j-primitive-collections</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.offbytwo</groupId>
+            <artifactId>docopt</artifactId>
```

Comment:
Unfortunately now we parse cmd args with `org.neo4j.helpers.Args` and do not use this library. Also import tool has some custom args syntax that docopt can't handle. It would be good to discuss the possibility to unify things.